src/repr/src/timestamp.rs

```diff
@@ -182,6 +198,54 @@ impl Timestamp {
     pub fn step_back(&self) -> Option<Self> {
         self.checked_sub(1)
     }
+
+    /// Rounds up the timestamp to the time of the next refresh, according to the given periodic
+    /// refresh schedule.
+    ///
+    /// Returns None if an overflow happens.
+    pub fn round_up(
+        &self,
+        RefreshEvery {
+            interval,
+            aligned_to,
+        }: &RefreshEvery,
+    ) -> Option<Self> {
```

Comment:
I noticed that the method to round up a timestamp according to a `RefreshSchedule` is defined on `RefreshSchedule`, but the method to round up a timestamp according to a `RefreshEvery` is defined on `Timestamp`. Could we move this logic to a `RefreshEvery::round_up_timestamp` method for consistency?

Reply:
Done